code/game/objects/structures/ECD.dm

```diff
@@ -0,0 +1,83 @@
+#define ECD_LOOSE 0
+#define ECD_BOLTED 1
+#define ECD_WELDED 2
+
+/obj/structure/ecd
```

Comment:
What is the point of this object? It doesn't seem to do anything except showing messages / playing sounds for what I'm seeing